src/usb_moded-configfs.c

```diff
@@ -823,6 +828,7 @@ configfs_init(void)
     configfs_register_function(FUNCTION_MTP);
 
     /* Prep: developer_mode */
+    configfs_register_function(FUNCTION_NCM);
     configfs_register_function(FUNCTION_RNDIS);
     if( (text = mac_read_mac()) ) {
         configfs_write_file(RNDIS_CTRL_ETHADDR, text);
```

Comment:
Should there be writing of mac address also for ncm? Based on quick look some kernel code seems to support.